app/code/Magento/InventoryConfiguration/Model/StockItemConfigurationInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryConfiguration\Model;
+
+/**
+ * Interface StockConfigurationInterface
+ *
+ * @api
+ */
+interface StockItemConfigurationInterface
+{
+    public function execute(string $sku, int $stockId, float $qtyWithReservation, bool $isSalable): bool;
```

Comment:
Currently, this interface looks strange for me.

If it's `StockItemConfigurationInterface` as its called, why does it accept
`float $qtyWithReservation, bool $isSalable` ?